modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: PROCEDURE
+
+[id="odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view_{context}"]
+= Customizing a Developer catalog or its sub-catalogs using the form view
+
+You can customize a developer catalog by using the form view in the Web Console.
+
+.Prerequisites
+
+* An Open Shift web console session by a cluster administrator.
+
+.Procedure
+
+. In the *Administrator* perspective, navigate to *Administration* -> *Cluster Settings*.
+. Select the *Configuration* tab and click the *Console (operator.openshift.io)* resource.
+. Click *Actions* -> *Customize*.
+. In the respective section, enable or disable the items in the list.
++
+.Verification
+After you have customized the developer catalog, your changes are automatically saved in the system and will take effect in the browser after a refresh.
++
+image::odc_customizing_developer_catalog.png[]
+
+[NOTE]
+====
+You can use a similar procedure to customize Web UI items such as, Quick starts, Cluster roles, and Actions.
```

Comment:
```suggestion
You can use a similar procedure to customize Web UI items such as Quick starts, Cluster roles, and Actions.
```

Reply:
Fixed. Thank you for such a detailed review @deerskindoll. I appreciate the amount of good feedback I have received on this PR. Will help me improve in the future.